File: cascadingFailuresTaskMigration_python/input/robot.py
from dataclasses import dataclass, field
from typing import List


@dataclass
class Robot:
    robot_id: int = 0
    capacity: float = 0.0
    load: float = 0.0
    tasks_list: List = field(default_factory=list)
    group_id: int = 0
    fault_a: float = 0.0
    fault_o: float = 0.0
    fault_s: float = 0.0

File: cascadingFailuresTaskMigration/src/main/java/input/Robot.java
package input;

import lombok.Data;

import java.util.List;

@Data
public class Robot {
    private int robotId;
    private double capacity;
    private double  load;
    private List<Task> tasksList;
    private int groupId;
    private double faultA;
    private double faultO;
    private double faultS;
}
